packages/lib/errors.ts

```diff
@@ -11,13 +11,17 @@ export function getErrorFromUnknown(cause: unknown): Error & { statusCode?: numb
 }
 
 export function handleErrorsJson(response: Response) {
+  if (response.headers.get("content-encoding") === "gzip") {
+    return response.text();
+  }
```

Comment:
When microsoft api graph request it's to large it gets here as gzip compressed. We should then use response.text(). It shouldn't affect other services.